src/reanimated2/threads.ts

```diff
@@ -23,6 +23,18 @@ export function runOnUI<A extends any[], R>(
   };
 }
 
+if (__DEV__) {
+  try {
+    runOnUI(() => {
+      'worklet';
+    })();
```

Comment:
Most likely `()` here is not necessary.
```suggestion
    });
```

Reply:
Indeed, it wasn't. Removed in d059e4a4704fddc4287abe2453bd3cec3cd6f927